Gemfile

```diff
@@ -54,10 +54,10 @@ gem '3scale_time_range', '0.0.6'
 gem 'statsd-ruby', require: false
 
 # Sidekiq
-gem 'sidekiq', '~> 6.4.0', require: %w[sidekiq sidekiq/web]
-gem 'sidekiq-batch'
+gem 'sidekiq', '~> 7', require: %w[sidekiq sidekiq/web]
+gem 'sidekiq-batch', github: '3scale/sidekiq-batch', branch: 'redis-client'
```

Comment:
this one, maybe add some code comment as to why we need a custom version so that later it removes from the guesswork when we try to upgrade

Reply:
I'll do